### ISSUE TITLE
Wyze Doorbell Glitching

### ISSUE THREAD

**Issue by cheme75** [mrlt8/docker-wyze-bridge#457]
Got bridge running on lan mode on laptop, all had been ok, db looks fine in wyze app but keep getting recurring disconnects, other cams look fine on bridge and via scrypted in home app - just the doorbell. I rebooted the DB and commented out the audio but no improvement - keeps disconnecting:
2022/06/30 12:40:55 [RTSP][FRONT-DOOR] ✅ '/front-door' stream is UP! (3/3)
2022/06/30 12:40:57 [RTSP][FRONT-DOOR] 📖 New client reading 
2022/06/30 12:41:26 [RTSP][FRONT-DOOR] ❌ '/front-door' stream is down
2022/06/30 12:41:26 [RTSP][FRONT-DOOR] 📕 Client stopped reading
2022/06/30 12:41:27 [Front-Door] AV_ER_SESSION_CLOSE_BY_REMOTE
2022/06/30 12:41:27 [WyzeBridge] 🎉 Connecting to WyzeCam Doorbell - Front-Door on 192.168.1.155 (1/3)
2022/06/30 12:41:30 [Front-Door] 📡 Getting 180kb/s HD stream (20fps) via LAN mode (WiFi: 81%) FW: 4.25.1.060 🔒 (DTLS) (2/3)
2022/06/30 12:41:30 [Front-Door] WARNING: Skipping smaller frame at start of stream (frame_size=4)
2022/06/30 12:41:31 [RTSP][FRONT-DOOR] ✅ '/front-door' stream is UP! (3/3)
2022/06/30 12:41:31 [RTSP][FRONT-DOOR] ❌ '/front-door' stream is down
2022/06/30 12:41:32 [Front-Door] AV_ER_SESSION_CLOSE_BY_REMOTE
2022/06/30 12:41:32 [WyzeBridge] 🎉 Connecting to WyzeCam Doorbell - Front-Door on 192.168.1.155 (1/3)
2022/06/30 12:41:35 [Front-Door] 📡 Getting 180kb/s HD stream (20fps) via LAN mode (WiFi: 75%) FW: 4.25.1.060 🔒 (DTLS) (2/3)
2022/06/30 12:41:35 [Front-Door] WARNING: Skipping smaller frame at start of stream (frame_size=4)
Cannot determine format of input stream 0:0 after EOF
Error marking filters as finished
2022/06/30 12:41:35 [Front-Door] AV_ER_SESSION_CLOSE_BY_REMOTE

and on and on...

**Comment by mrlt8** [comment]
Hmm that is weird. Do you see the same behavior on your other bridge with the networking issues?

**Comment by cheme75** [comment]
> Hmm that is weird. Do you see the same behavior on your other bridge with the networking issues?

I think it was due to the other bridge - I forgot i had a start task set to run each morning. It was forced to lan mode and was failing repeatedly - not sure why other cams were ok. but i shutdown that one and now this one looks steady. A couple time outs but it has connected and is ok. I'll up the timeout to 30 before next update which should fix those few hiccups. still haven't found anything on the other pc.

**Comment by cheme75** [comment]
@mrlt8 new glitch: only doorbell will not display in the web ui - keeps getting error, other 2 v2 cams look fine...
2022/06/30 16:59:27 [RTSP][ENTRY] 📖 New client reading 

2022/06/30 16:59:27 [RTSP][PATIO] 📖 New client reading 

2022/06/30 16:59:28 [RTSP][PATIO] 📕 Client stopped reading

2022/06/30 16:59:28 [WyzeBridge] 172.18.0.1 - - [30/Jun/2022 16:59:28] "GET /snapshot/patio.jpg HTTP/1.1" 200 -

2022/06/30 16:59:29 [RTSP][ENTRY] 📕 Client stopped reading

2022/06/30 16:59:29 [WyzeBridge] 172.18.0.1 - - [30/Jun/2022 16:59:29] "GET /snapshot/entry.jpg HTTP/1.1" 200 -

2022/06/30 16:59:29 [WyzeBridge] 172.18.0.1 - - [30/Jun/2022 16:59:29] "GET /snapshot/front-door.jpg HTTP/1.1" 404 -

2022/06/30 16:59:29 [WyzeBridge] 172.18.0.1 - - [30/Jun/2022 16:59:29] "GET /snapshot/front-door.jpg HTTP/1.1" 404 -

**Comment by mrlt8** [comment]
Is the doorbell online/connected?

**Comment by cheme75** [comment]
> Is the doorbell online/connected?

Yes, looks and behaves normally in the wyze app

**Comment by cheme75** [comment]
Nope. It’s not showing up. I think I just lost the v2 that I’m rotating. Only seeing the standard. V2. No longer seeing either rotated cam, but can’t get back to bridge machine right now. I’ll stop rotating these tomorrow to test it out. First I’ll switch it back to the original pc to see if it’s pc related.

**Comment by cheme75** [comment]
@mrlt8 - before I hit the sack, I switched back to bridge on old machine. Still goes to relay mode but all 3 cams working fine - rotation is good. Can’t see much at night but all are running normal. Will revisit the laptop and lan mode tomorrow.

**Comment by cheme75** [comment]
@mrlt8 - update: shutdown bridge on desktop, rebooted laptop, restarted docker, pulled latest, now running without error in lan mode, web ui and browsing look fine. Rtsp streams look fine in homekit via scrypted, just seems to hiccup on getting snapshot for the doorbell cam once in a while - web ui looks fine on other devices and streams look ok.  Saw a few errors earlier in log will post later. CPU on the laptop is cooking at well over 90%.

**Comment by cheme75** [comment]
@mrlt8 - here's the log from yesterday when it was glitching - note that the 172 subnet is not my lan which is 192. The 172 subnet looks to be for the wsl v-ethernet adapter. which is similar to how its setup on the desktop pc - has two on two different 172 subnets, v-ethernet wsl adapter and v-ethernet default switch - anyway after rebooting laptop these errors disappeared. however due to high cpu usage, i went back to bridge on desktop and seeing no errors just still stuck in relay mode:

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET / HTTP/1.1" 200 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /favicon.ico HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /apple-touch-icon.png HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /favicon.ico HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET /apple-touch-icon.png HTTP/1.1" 404 -

2022/07/01 19:18:02 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:02] "GET / HTTP/1.1" 200 -

2022/07/01 19:18:04 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:04] "GET /favicon.ico HTTP/1.1" 404 -

2022/07/01 19:18:04 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:04] "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 -

2022/07/01 19:18:04 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:04] "GET / HTTP/1.1" 200 -

2022/07/01 19:18:04 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:04] "GET /apple-touch-icon.png HTTP/1.1" 404 -

2022/07/01 19:18:17 [RTSP][ENTRY] 📖 New client reading 

2022/07/01 19:18:17 [RTSP][FRONT-DOOR] 📖 New client reading 

2022/07/01 19:18:17 [RTSP][PATIO] 📖 New client reading 

2022/07/01 19:18:18 [RTSP][ENTRY] 📕 Client stopped reading

2022/07/01 19:18:18 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:18] "GET /snapshot/entry.jpg HTTP/1.1" 200 -

2022/07/01 19:18:18 [RTSP][FRONT-DOOR] 📕 Client stopped reading

2022/07/01 19:18:18 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:18] "GET /snapshot/front-door.jpg HTTP/1.1" 200 -

2022/07/01 19:18:18 [RTSP][ENTRY] 📖 New client reading 

2022/07/01 19:18:18 [RTSP][FRONT-DOOR] 📖 New client reading 

2022/07/01 19:18:18 [RTSP][PATIO] 📕 Client stopped reading

2022/07/01 19:18:18 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:18] "GET /snapshot/patio.jpg HTTP/1.1" 200 -

2022/07/01 19:18:19 [RTSP][PATIO] 📖 New client reading 

2022/07/01 19:18:20 [RTSP][ENTRY] 📕 Client stopped reading

2022/07/01 19:18:21 [WyzeBridge] 172.18.0.1 - - [01/Jul/2022 19:18:21] "GET /snapshot/entry.jpg HTTP/1.1" 200 -

2022/07/01 19:18:21 [RTSP][PATIO] 📕 Client stopped reading

**Comment by mrlt8** [comment]
That's from the web-ui. 

You can ignore the 404s, that's just iOS/safari. 

The other `200` messages are from the snapshot preview auto updating.

**Comment by cheme75** [comment]
I’m not keeping web ui open very much so it’s not a big deal other than being blank when I do want to view it. Plus I also got failed snapshots on scrypted and at times on home app when I used bridge on the laptop and at times failed to load stream. I only go into scrypted to tweak motion detection settings. Home app I’m only opening just to have a look or when I see a motion alert. Nothing is constantly requesting snapshots unless it happens in background regardless. After switching back to desktop only very minor delays, no missed snapshots, streams pop open. Maybe WiFi congestion for laptop adapter? The desktop has a 1gb wired connection. Also with laptop cpu cooking at over 90% I figure I’m safer using desktop using cpu at 40%. I’ll live with relay mode for the better performance and reliability. Not sure why laptop is being hit so hard, will have to check on other background tasks I might be able to stop. Win 10 is a bit of a pig.
